Marcio/Notification docs (#1612)
cloudflare/cloudflare-docs: products/fundamentals/src/content/notifications/configure-create/edit-notification.md

---
order: 1
title: Edit Notifications and webhooks
pcx-content: how-to
---

> # How to edit a Notification

Reviewer: ```suggestion
# Edit a Notification
```
(for consistency with `Create a Notification`)

1. Sign in to your [Cloudflare dashboard](https://dash.cloudflare.com/login).
1. Click **Notifications** on the upper right corner.
1. Click **Edit** on the Notification you want to modify.
1. Make changes as needed.

> 1. You can also opt to disable Notifications instead of deleting them. 

Reviewer: Same as above, I would describe the entire procedure and give a success message in the end.
Also, what's the purpose of including the last step here? Enabling or deleting a notification doesn't seem to be part of the `Edit` action, which takes you to a separate page. Should we call this out elsewhere?

Author: Done.

## Webhooks to external services

> There are a variety of services you can connect to using webhooks, in order to receive Notifications from your Cloudflare account. Here are links to some of the most requested services you can connect your Cloudflare account to:

Reviewer: ```suggestion
There are a variety of services you can connect to Cloudflare using webhooks, in order to receive Notifications from your Cloudflare account. Here are some of the most popular services you can connect to your Cloudflare account:
```
nit
Also `requested` made me quickly go `by whom?`, so I proposed `popular`

Author: Added

* [Jira](https://developer.atlassian.com/server/jira/platform/webhooks/) 
* [Slack](https://api.slack.com/messaging/webhooks)
* [GChat](https://developers.google.com/chat/how-tos/webhooks)
* [DataDog](https://docs.datadoghq.com/developers/guide/calling-on-datadog-s-api-with-the-webhooks-integration/)
* [OpsGenie](https://docs.opsgenie.com/docs/integration-api)

> After you are done, you can set up webhooks in your [Cloudflare dashboard](https://dash.cloudflare.com/login):

Reviewer: what's the procedure you're referring to with `done`?

Author: the external service. I've updated the text to reflect this.

1. Go to **Notifications**. 

> 1. On the left of your dashboard, click **Destinations**.

Reviewer: Very generally speaking, I always try to start these steps with an action, `Click **Destinations** on the left.` (also, could we be a bit more specific? I think it's me not having an enterprise account but I don't see the button)

Author: Yup. It's invisible unless you have the proper account level. I'll rewrite it

> 1. You will see an area called **Webhooks**. Click the button **Create**.

Reviewer: ```suggestion
1. You will see a card called **Webhooks**. Click **Create**.
```
(not sure what you meant by 'area', I usually refer to the boxes as 'cards' at least on the Teams dash)

Author: Change it

1. Give your webhook a name so you can identify it later.

> 1. Fill in the URL you set up above, in the third party service you want to connect your Cloudflare account to — for example, Slack.

Reviewer: ```suggestion
1. Enter the URL you set up above, in the field corresponding to the third-party service you want to connect to your Cloudflare account.
```

Author: Changed

> 1. If needed, insert the **Secret** in its field. Secrets are how webhooks are encrypted and vary according to the service you are trying to use.

Reviewer: ```suggestion
1. If needed, insert the **Secret**. Secrets are how webhooks are encrypted and vary according to the service you are connecting to Cloudflare.
```

Author: Changed

1. Click **Notifications** on the upper right corner.
1. Click **Edit** on the Notification you want to modify.
1. Make changes as needed and click **Save**.

This will take you back to your list of Notifications.

<Aside type="note" header="Note">

You can also disable and/or delete Notifications. To disable a notification, click the **Enabled** toggle. To delete it, click **Delete**.

</Aside>

## Edit a PagerDuty connected service

You can only disconnect PagerDuty services.

1. Sign in to your [Cloudflare dashboard](https://dash.cloudflare.com/login).
1. Go to **Notifications**.
1. Click **Destinations** on the left side of your dashboard.
1. In the **Connected notification services** card, click **View** on the PagerDuty you want to disconnect.
1. Click **Disconnect**.
1. Select **Confirm**.

## Edit webhooks

You can only edit the name of webhooks and/or delete them.

1. Sign in to your [Cloudflare dashboard](https://dash.cloudflare.com/login).
1. Go to **Notifications**.
1. Click **Destinations** on the left side of your dashboard.
1. In the **Webhooks** card, click **Edit** on the webhook you want to edit.

> 1. Change the name as you see fit.

Reviewer 2: This line didn't change (just a heads-up -- you may have chosen not to accept the suggestion 🙂 ).

Author: My bad
